index.html

```diff
@@ -8,15 +8,7 @@
 </head>
 
 <body>
-    <main id="app">
-        <h1>🍿 자판기 🍿</h1>
-        <nav>
-            <a id="item-manage-tab-button" class="nav-tab-button selected" href="#item-manage">상품 관리</a>
-            <a id="coin-recharge-tab-button" class="nav-tab-button" href="#coin-recharge">잔돈 충전</a>
-            <a id="item-purchase-tab-button" class="nav-tab-button" href="#item-purchase">상품 구매</a>
-        </nav>
-        <section id="tab-content"></section>
-    </main>
+    <main id="app"></main>
 </body>
 
 </html>
```

Comment:
인덴트가 안맞네요. html 파일 관련된 프리티어 설정이 안되어있을까요?
```suggestion
  </body>

</html>
```

Reply:
설정한 줄 알았는데, 기본 포매터가 HTML 언어 기능으로 되어 있었네요..ㅎㅎ 수정했습니다!